tests/test_stt.py

```diff
@@ -35,7 +35,7 @@ def read_mp3_file(filename: str) -> rtc.AudioFrame:
     return agents.utils.merge_frames(frames)
 
 
-RECOGNIZE_STT = [deepgram.STT(), google.STT(), openai.STT(), azure.STT()]
+RECOGNIZE_STT = [deepgram.STT(), google.STT(), openai.STT()]
```

Comment:
we removed azure STT from tests?

Reply:
Only for the recognize method that we don't support. We only support streaming for it and it is correctly being testing at the end of the file

Reply:
got it. interesting they only support streaming mode.

Reply:
They do support it: https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/d7e66d66850ecee97d4e8c01950a100b441364d7/samples/python/console/speech_sample.py#L42

Fwiw recognize method is never used when streaming is supported. I will add a ticket for it but this is low priority